abc.templates/base-workflows/contents/workflows/reusable-guardian-plan.yml

```diff
@@ -121,7 +112,7 @@ jobs:
 
   plan:
     if: |
-      github.event_name == 'pull_request_target' && needs.init.outputs.directories != '[]'
+      needs.init.outputs.directories != '[]'
```

Comment:
i think wee need to keep this but make it pull_request, this was added to support the mrege_group event i believe